en/components/time_picker.md

```diff
@@ -0,0 +1,211 @@
+</div>
+<div class="divider--half"></div>
+
+### API Summary
+The following tables summarize some of the useful **igx-time-picker** component inputs, outputs and methods.
+
+#### Inputs
+The following inputs are available in the **igx-time-picker** component:
+
+| Name | Type | Description |
+| :--- | :--- | :--- |
+| `okButtonLabel` | string | Renders OK button with custom text, which commits the selected time, and fill the timePicker input. By default `okButtonLabel` is set to `OK`.|
+| `cancelButtonLabel` | string | Renders cancel button with custom text, which closes the dialog. By default `cancelButtonLabel` is set to `Cancel`. |
+| `value` | Date | Gets/Sets the value of the timePicker. |
+| `isDisabled` | boolean | Disable the timePicker. |
+| `itemsDelta`| object | Gets/Sets the delta by which hour and minute items would be displayed. By default `itemsDelta` is set to `{hours:1, minutes:1}`. Note! This cannot be set run time. |
+| `minValue` | string | Gets/Sets minimum value. It should follow the `format` of the timePicker. |
+| `maxValue` | string | Gets/Sets maximum value. It should follow the `format` of the timePicker. |
+| `vertical` | boolean | Gets/Sets the orientation of the timePicker. By default `vertical` is set to `false`. |
+| `isSpinLoop` | boolean | Determines the spin behavior. If set to true the hours and minutes spinning will wrap around. By default `isSpinLoop` is set to true. Note! This cannot be set run time. |
+| `format` | string | Gets/Sets format of time while timePicker has no focus. By default `format` is set to `hh:mm tt`. Note! This cannot be set run time.<br>List of time-flags:<br> `h` : hours field in 12-hours format without leading zero<br>`hh` : hours field in 12-hours format with leading zero<br>`H` : hours field in 24-hours format without leading zero<br>`HH` : hours field in 24-hours format with leading zero<br>`m` : minutes field without leading zero<br>`mm` : minutes field with leading zero<br>`tt` : 2 characters of string which represents AM/PM field |
+
+<div class="divider--half"></div>
+
+#### Outputs
+The following outputs are available in the **igx-time-picker** component:
+
+| Name | Description |
+| :--- | :--- |
+| `onValueChanged` | Emitted when selection is made. The event contains the selected value. Returns `{oldValue: Date, newValue: Date}` |
+| `onValidationFailed` | Emitted when an invalid value is being set. Returns `{timePicker: any, currentValue: Date, setThroughUI: boolean}` |
+| `onOpen` | Emitted when a timePicker is being opened. |
+
+#### Methods
+The following methods are available in the **igx-time-picker** component:
+
+| Signature | Return Type | Description |
+| :--- | :--- | :--- |
+| `okButtonClick()` | `void` | If current value is valid selects it and closes the dialog. **[Otherwise this should return a value that it fails]** |
```

Comment:
Please take a look at my comment here. 

I made some minor changes to the document, but have not copied them over in the JP view.